### ISSUE TITLE
We should depend on luci-lib-jsonc instead of luci-lib-json

### ISSUE THREAD

**Issue by G10h4ck** [libremesh/lime-packages#475]
Seems luci-lib-jsonc is more standard compliant regarding JSON format, and other OpenWrt components already depends on it as you can see from the following paste

```
root@LiMe-Zumba:~# opkg whatdepends luci-lib-json
Root set:
  luci-lib-json
What depends on root set
        luci-lib-libremap 0.1.9 depends on luci-lib-json
        ubus-lime-metrics 2018-06-04-1528040806 depends on luci-lib-json
        ubus-lime-openairview 2019-02-16-1550326422     depends on luci-lib-json
        libremap-agent 0.1.9    depends on luci-lib-libremap
root@LiMe-Zumba:~# opkg whatdepends luci-lib-jsonc
Root set:
  luci-lib-jsonc
What depends on root set
        luci-base git-19.046.40869-30d9bc0-1    depends on luci-lib-jsonc
        luci-lib-httpclient git-19.046.40869-30d9bc0-1  depends on luci-base
        luci-lib-httpprotoutils git-19.046.40869-30d9bc0-1    depends on luci-base
root@LiMe-Zumba:~#
```

**Comment by ilario** [comment]
https://github.com/libremesh/lime-packages/search?q=luci-lib-json

**Comment by aparcar** [comment]
sound good, you fancy a PR?

**Comment by ilario** [comment]
Currently `luci-lib-json` is mentioned [in the Makefile of 5 packages](https://github.com/libremesh/lime-packages/search?q=luci-lib-json) but [it gets imported by 9 packages](https://github.com/libremesh/lime-packages/search?q=%22luci.json%22) (???) and actually used by 8 ([lime-fbw](https://github.com/libremesh/lime-packages/blob/fd543f9b03b5be054012f360ac1ab7e22d4c3d2a/packages/ubus-lime-fbw/files/usr/libexec/daemon/lime-fbw) includes it but then does not use it).
Could you explain how the migration to `luci-lib-jsonc` should be performed? How much are the two syntax different?

**Comment by spiccinini** [comment]
To do the migration the changes needed are:
* Replace `require luci.json` by `require luci.jsonc`
* Replace `json.encode` by `json.stringify`
* Replace `json.decode` by `json.parse`
* Replace the dependencies at the makefile

**Comment by spiccinini** [comment]
Closed by #696
